sdk-workflows/src/main/java/io/dapr/workflows/client/WorkflowStatusQuery.java

```diff
@@ -0,0 +1,212 @@
+   */
+  public WorkflowStatusQuery setTaskHubNames(@Nullable List<String> taskHubNames) {
+    this.taskHubNames = taskHubNames;
+    return this;
+  }
+
+  /**
+   * Sets the continuation token used to continue paging through orchestration metadata results.
+   * 
+   * <p>This should always be the continuation token value from the previous query's
+   * {@link WorkflowStatusQueryResult} result.
+   *
+   * @param continuationToken the continuation token from the previous query
+   * @return this query object
+   */
+  public WorkflowStatusQuery setContinuationToken(@Nullable String continuationToken) {
+    this.continuationToken = continuationToken;
+    return this;
+  }
+
+  /**
+   * Include orchestration metadata records with the specified instance ID prefix.
+   *
+   * <p>For example, if there are three orchestration instances in the metadata store with IDs "Foo", "Bar", and "Baz",
+   * specifying a prefix value of "B" will exclude "Foo" since its ID doesn't start with "B".
+   *
+   * @param instanceIdPrefix the instance ID prefix filter value
+   * @return this query object
+   */
+  public WorkflowStatusQuery setInstanceIdPrefix(@Nullable String instanceIdPrefix) {
+    this.instanceIdPrefix = instanceIdPrefix;
+    return this;
+  }
+
+  /**
+   * Sets whether to fetch orchestration inputs, outputs, and custom status values. The default value is {@code false}.
+   *
+   * @param fetchInputsAndOutputs {@code true} to fetch orchestration inputs, outputs, and custom status values,
+   *                              otherwise {@code false}
+   * @return this query object
+   */
+  public WorkflowStatusQuery setFetchInputsAndOutputs(boolean fetchInputsAndOutputs) {
+    this.fetchInputsAndOutputs = fetchInputsAndOutputs;
+    return this;
+  }
+
+  /**
+   * Gets the configured runtime status filter or {@code null} if none was configured.
+   * @return the configured runtime status filter as a list of values or {@code null} if none was configured
+   */
+  public List<WorkflowRuntimeStatus> getRuntimeStatusList() {
+    return runtimeStatusList;
+  }
+
+  /**
+   * Gets the configured minimum orchestration creation time or {@code null} if none was configured.
+   * @return the configured minimum orchestration creation time or {@code null} if none was configured
+   */
+  @Nullable
+  public Instant getCreatedTimeFrom() {
+    return createdTimeFrom;
+  }
+
+  /**
+   * Gets the configured maximum orchestration creation time or {@code null} if none was configured.
+   * @return the configured maximum orchestration creation time or {@code null} if none was configured
+   */
+  @Nullable
+  public Instant getCreatedTimeTo() {
+    return createdTimeTo;
+  }
+
+  /**
+   * Gets the configured maximum number of records that can be returned by the query.
+   * @return the configured maximum number of records that can be returned by the query
+   */
+  public int getMaxInstanceCount() {
+    return maxInstanceCount;
```

Comment:
Same configuration in case this can be nullable

Reply:
Done